march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -213,20 +213,13 @@ void MarchHardwareInterface::read(const ros::Time& /* time */, const ros::Durati
         MarchHardwareInterface::ALPHA * joint_velocity + (1 - MarchHardwareInterface::ALPHA) * joint_velocity_[i];
 
     joint_effort_[i] = joint.getTorque();
-
-    ROS_DEBUG("Joint %s: read position %f", joint_names_[i].c_str(), joint_position_[i]);
   }
 
   this->updateIMotionCubeState();
 
   if (has_power_distribution_board_)
   {
     power_distribution_board_read_ = march_robot_.getPowerDistributionBoard();
-
-    if (!power_distribution_board_read_.getHighVoltage().getHighVoltageEnabled())
-    {
-      ROS_WARN_THROTTLE(10, "All-High-Voltage disabled");
-    }
   }
```

Comment:
I think this is a very useful statement if we want to run with powerdistributionboard again

Reply:
True, however it also printed here https://github.com/project-march/hardware-interface/blob/c5fdaffece8cf0b8b137124faec1034743c7bcce/march_hardware_interface/src/march_hardware_interface.cpp#L327-L330

I found the above location a better one